packages/core/src/actions/public/account-factory/get-account-factory-address.ts

```diff
@@ -0,0 +1,42 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate'
+
+import {
+  VersionControlQueryClient,
+  VersionControlTypes,
+} from '../../../codegen/abstract'
+import { versionControlModuleToAddress } from '../../../utils/version-control/version-control-module-to-address'
+
+const ACCOUNT_FACTORY_MODULE_NAME = 'account-factory'
+
+export async function getAccountFactoryAddress(
+  cosmWasmClient: CosmWasmClient,
+  versionControlAddress: string,
+  version?: string,
+) {
+  const registryClient = new VersionControlQueryClient(
+    cosmWasmClient,
+    versionControlAddress,
+  )
```

Comment:
To confirm - We refrain from passing the `VersionControlQueryClient` explicitly instead of the CWClient and the VC Address because we don't want to tie the impl to the ts-codegen version?